From d19671fa23cfdd8775ab1e710c244603ac2c9726 Mon Sep 17 00:00:00 2001
From: John Howard <howardjohn@google.com>
Date: Mon, 16 May 2022 19:49:07 -0700
Subject: tf: disable wait on YAML (#38939)

If we do this long term we should probably remove the option entirely.
For now, just hack it out to see the impact.
---
 pkg/test/framework/resource/config/apply/option.go | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/pkg/test/framework/resource/config/apply/option.go b/pkg/test/framework/resource/config/apply/option.go
index a9a3bb5201..92261bf9fc 100644
--- a/pkg/test/framework/resource/config/apply/option.go
+++ b/pkg/test/framework/resource/config/apply/option.go
@@ -41,6 +41,7 @@ func (f OptionFunc) Set(opts *Options) {
 })
 
 // Wait configures the Options to wait for configuration to propagate.
+// This method currently does nothing, due to https://github.com/istio/istio/issues/37148
 var Wait Option = OptionFunc(func(opts *Options) {
-	opts.Wait = true
+	// opts.Wait = true
 })
-- 
2.35.3

